src/OrchardCore/OrchardCore.Users.Abstractions/Handlers/IUserHandler.cs

```diff
@@ -0,0 +1,9 @@
+using System.Threading.Tasks;
+
+namespace OrchardCore.Users.Handlers
+{
+    public interface IUserHandler
```

Comment:
I like it and the way you added it in a service.

But it might be better to add it in the `UserManager` because the UserService only adds error management.

Also can you chance this interface to `IUserCreatedEventHandler` ?

I think keeping a single method in an interface is nice because it makes implementations simpler when we only need to handle one method. We could do this with O1 and the event bus, but we don't have it anymore here.

Reply:
UserManager is in Microsoft.AspNet.Identity

Reply:
Ah, then in the store, or whatever class implements this method using yessql.

Also you made a mistake in the interface name: IUserCreatedEventHandler instead of IUserCreatedEventDisplay